app/Http/Controllers/Account/AcceptanceController.php

```diff
@@ -323,7 +325,7 @@ public function store(Request $request, $id)
                 Storage::put('private_uploads/eula-pdfs/' .$pdf_filename, $pdf->output());
             }
 
-            $acceptance->decline($sig_filename);
+            $acceptance->decline($sig_filename, $request->input('note'));
```

Comment:
I think we need to pass `$request->input('note')` along for acceptance too (on line 241).